CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 
 - `[docs]` Updated documentation for the `--runTestsByPath` CLI command ([#14004](https://github.com/facebook/jest/pull/14004))
 - `[docs]` Updated documentation regarding the synchronous fallback when asynchronous code transforms are unavailable ([#14056](https://github.com/facebook/jest/pull/14056))
+- `[jest-snapshot]` Insert direct link to docs in snapshots instead of using URL shortener ([#14081](https://github.com/facebook/jest/pull/14081))
```

Comment:
Ups.. Sorry, I managed missed that somehow.

```suggestion
- `[jest-snapshot]` [**BREAKING**] Insert direct link to docs in snapshots instead of using URL shortener ([#14081](https://github.com/facebook/jest/pull/14081))
```